renovate.json5

```diff
@@ -0,0 +1,82 @@
+{
+  extends: [
+    'config:base'
+  ],
+
+  /**
+   * Skip any package file whose path matches one of these. Can be string or glob pattern
+   */
+  includePaths: [
+    'package.json',
+  ],
+
+  /**
+   * Only submit PRs to these branches, we will manually backport PRs for now
+   */
+  baseBranches: ['master'],
+
+  /**
+   * Labels added to PRs opened by renovate
+   */
+  labels: [
+    'renovate',
+    'release_note:skip',
+    ':Operations',
+  ],
+
+  /**
+   * Enable creation of a "Master Issue" within the repository. This
+   * Master Issue is akin to a mini dashboard and contains a list of all
+   * PRs pending, open, closed (unmerged) or in error.
+   */
+  masterIssue: true,
+
+  npm: {
+    /**
+     * This deletes and re-creates the lock file, which we will only want
+     * to turn on once we've updated all our deps
+     */
+    lockFileMaintenance: { enabled: false },
```

Comment:
I think we will never want this enabled right? Or do you think renovate will support well our yarn lock file generation logic with the workspaces + every package.json across the repo?

Reply:
I think it will work once we've got the packages up to date.